content/blog/cdf-intro-uber-crane/index.md

```diff
@@ -32,20 +32,19 @@ These two things significantly reduced the time to spin up a new zone, from mont
 
 ### Using OpenFABR CDF
 
-OpenFABR CDF is a framework for building IaC using imperative programming languages like TypeScript, Python, or Golang. It's the glue that gives IaC the full software lifecycle support.
+OpenFABR CDF is a framework for building IaC using imperative programming languages like TypeScript, Python, or Golang. It is the glue that gives IaC the full software lifecycle support. In this case, the likes of Terraform CDK, paired with Golang as the programme language of the choice, would fit Uber's infrastructure implementation as described in the post.
 
-- Pick either CDK Terraform or Pulumi as the IaC runtime and language. Because Uber needs to support hybrid and multi-cloud AWS CDK out of the mix. We'll pick CDK TF and Golang.
-- We'll have to build custom providers for the on-prem custom infrastructure (see [Writing Custom Terraform Providers](https://www.hashicorp.com/blog/writing-custom-terraform-providers)). We assume they have the needed control plane APIs.
-- Develop contructs to represent the various infrastructure abstractions. Publish them as a Go package(s) into your standard Go package repository. They can then be imported as dependencies in other IaC codebase.
-- Transitive dependencies can cause the circular dependencies problem as mentioned earlier, however some dependency-focused static code analysis tools could pick out abnormal inter-layer dependencies easily, similar to how [GitHub Dependabot](https://github.com/dependabot) works.
+With imperative Iac, building blocks (`constructs` as described in some frameworks) can be developed, tested and then published just as normal software packages, perhaps in *Golang* as the post mentioned. Once they are in Go package repository, they can be imported as dependencies and used in other IaC codebase.
```

Comment:
I think the previous paragraphs are too descriptive, some of which will also be repeated in the example section. Here I opt to make high level comments.